ovirt/resource_ovirt_vm.go

```diff
@@ -636,6 +684,18 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	// update the VM with the auto pinning
+	if isAutoPinning {
+		autoPinningPolicy := ovirtsdk4.AutoPinningPolicy(fmt.Sprint(d.Get("auto_pinning_policy")))
+		if autoPinningPolicy != ovirtsdk4.AUTOPINNINGPOLICY_DISABLED {
+			log.Printf("[DEBUG] Setting Auto Pinning Policy to VM (%s).", d.Id())
+			err := ovirtUpdateAutoPinning(d.Id(), autoPinningPolicy, meta)
+			if err != nil {
+				log.Printf("[DEBUG] Updating the VM (%s) with auto pinning policy failed!", d.Id())
```

Comment:
Shouldn't we fail here? If we don't fail Terraform will think everything is OK and will not try again on the next run.

Reply:
The questions is whether we want to make this error as fatal, or we can live with continuing (and failing here), letting the logs tell the problem and not having the VMs configured as expected.